src/rust/rust-proto/tests/test_utils/strategies.rs

```diff
@@ -82,13 +82,44 @@ pub mod common {
     use super::*;
     prop_compose! {
         pub fn edge_names()(
-            name in string_not_empty()
+            name in "[a-z]+(_[a-z]+)*"
         ) -> native::EdgeName {
+            let name = name.chars().take(32).collect::<String>();
             native::EdgeName{
                 value: name
             }
         }
     }
+
+    prop_compose! {
+        pub fn property_names()(
+            name in "[a-z]+(_[a-z]+)*"
+        ) -> native::PropertyName {
+            let name = name.chars().take(32).collect::<String>();
+            native::PropertyName {
+                value: name
+            }
+        }
+    }
+
+    prop_compose! {
+        pub fn node_types()(
+            name in "([A-Z][a-z]+)+"
```

Comment:
i feel like one of these +'s is extraneous

Reply:
nope, it's a repeating pattern